ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,179 @@
+# Alzadaal's Legacy
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# SILKIE #
+#~~~~~~~~#
+
+# -ii 77ED 7762 7768 7769 776A
+
+0.0 "--sync--" sync / 00:0839::The Trial of Knowledge is sealed off/ window 0,1
+12.2 "--sync--" sync / 14:[^:]*:Silkie:7759:/ window 15,15
+14.9 "Fizzling Suds" sync / 1[56]:[^:]*:Silkie:7759:/
+21.0 "Soap's Up" sync / 1[56]:[^:]*:Silkie:775A:/
+22.0 "Fizzling Duster" sync / 1[56]:[^:]*:Silkie:775D:/
+32.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+38.4 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+43.5 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+50.1 "Squeaky Clean" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+51.6 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+53.3 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+54.8 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+61.9 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+64.0 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+69.0 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+69.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:7760:/
```

Comment:
This pattern is suspicious to me.  Is 7760 the damage and 775E the self-targeted ability? If so, add 7760 to `-ii` and remove it.

Reply:
Yea, 7760 is the damage applied after cast end. This ability is a bit weird. It's basically 1 second after Slippery Soap damage goes out, the boss stops moving/tracking the target, so depending on latency player could receive the damage early and be moving to dodge followup mechanic and the boss follows them still.